### PR TITLE
Restore link style

### PR DESCRIPTION
**Pull request by chris-gds**
## What?

Restoring underline from gem to indicate to a user this is a link so it doesn't rely on colour alone.

## Why?

Divergence from [gem](https://components.publishing.service.gov.uk/component-guide) & [WCAG 1.4.1 issue](https://www.w3.org/TR/WCAG20-TECHS/F73.html)

## Visuals

Before (left) / After (right)

Screenshot 2021-02-10 at 17 21 02

Screenshot 2021-02-10 at 17 21 50

## Anything else?

This PR is dependant on [Accordion Design updates PR](https://github.com/alphagov/govuk_publishing_components/pull/1884) being merged and released first.

### CODE REVIEW

**Comment by owenatgov** [comment]
You may've mentioned this at standup, sorry if you're having to repeat yourself: has this been given the ok by the covid team? You might get the end of a hairdryer if this suddenly changes without them knowing.

**Comment by chris-gds** [comment]
> You may've mentioned this at standup, sorry if you're having to repeat yourself: has this been given the ok by the covid team? You might get the end of a hairdryer if this suddenly changes without them knowing.

Not at all, it's a consideration that's needed.  There's a couple of pieces of work just before, this is getting things ready:

- Once the Design has been finalised and the PR has been approved then.. (we're here right now) 
- There's a wider update needed to GOV.UK advising of the Design change as the Accordion isn't just used on collections.
- There might be some feedback to consider.
- Once released to the gem this will mean that all implementations should be updated to absorb the Design changes intended.
- _Then_ this PR will aim to resolve an issue where the Design updates will not cascade as expected.  
(@DilwoarH - this would be good to have on your radar as something going on in the background atm)

**Comment by chris-gds** [comment]
Closing as [related PR](https://github.com/alphagov/collections/pull/2260) resolves issue.

**Comment by chris-gds** [comment]
Re-opening upon testing [PR #2660](https://github.com/alphagov/collections/pull/2260) resolves accordion divergence on this page: https://www.gov.uk/world/japan. However this PR is still needed to gain a link style update.

File: app/views/coronavirus_landing_page/components/shared/_section.html.erb
Comment (maxgds) [inline app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: Should we also remove the css that declared covid__list--accordion?

Comment (chris-gds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: Ah it doesn’t exist, I think it’s a stray tag.

Comment (maxgds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: I see it in master: https://github.com/alphagov/collections/blob/ad6ca58abf14712d5ed623e04bea32a72fd998e1/app/assets/stylesheets/views/_covid.scss line 145

Comment (chris-gds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: :see_no_evil: I see it now - removed

Comment (maxgds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: I think with the update you've lost the previous deletion from the file where .covid__accordion-link is declared.

**Comment by DilwoarH** [comment]
In the screenshot, it shows accordions styled differently but on the preview link it's still the same: https://govuk-collec-remove-acc-qb7jze.herokuapp.com/coronavirus

**Comment by chris-gds** [comment]
> In the screenshot, it shows accordions styled differently but on the preview link it's still the same: https://govuk-collec-remove-acc-qb7jze.herokuapp.com/coronavirus

Hey ya, ah yes I would imagine this would be as the gem release / dependant-a-bot has not been merged in just yet (this only happened today) this is lining things up.

File: app/views/coronavirus_landing_page/components/shared/_section.html.erb
Comment (chris-gds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: urgh - removed. Panic updating of course goes awry, on hindsight once re-opened I should have put into instantly into draft so I could wait for the release merge. #TIL

Comment (maxgds) [reply app/views/coronavirus_landing_page/components/shared/_section.html.erb:10]: No worries, glitches like this are what the process is here to spot. Like you say, let's wait for the release and double check visuals before approve/merge.

**Comment by chris-gds** [comment]
I've merged the dependabot update into this PR and updated app-level testing (we can close the dependabot PR after this)

Changes can now been seen below:
https://govuk-collec-remove-acc-qb7jze.herokuapp.com/coronavirus

**Review by maxgds — APPROVED** [review]

**Pull request merged**
